pkg/provisioner/ironic/ironic.go

```diff
@@ -177,12 +183,21 @@ func newProvisionerWithSettings(host metal3v1alpha1.BareMetalHost, bmcCreds bmc.
 	if err != nil {
 		return nil, err
 	}
+	// TODO(iurygregory): Update here if we want to consider a trusted CAFile for the bmc.
+	bmcTLSConf := clients.TLSConfig{
+		TrustedCAFile:      "",
+		InsecureSkipVerify: host.Spec.BMC.DisableCertificateVerification,
```

Comment:
(note: if it's something to be managed later then maybe it could have an hard-coded value?)